packages/cloud-cognitive/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -17,19 +16,112 @@ import { ComboButton, ComboButtonItem } from '..';
 
 import styles from './_combo-button.scss';
 
+const modifiers = {
+  'Is Delete': { isDelete: true },
+  'No Delete': { isDelete: false },
+  'Has Divider': { hasDivider: true },
+  'No Divider': { hasDivider: false },
+  'Item Disabled': { itemDisabled: true },
+  'Item Active': { itemDisabled: false },
+};
 export default {
   title: getStoryTitle(ComboButton.displayName),
   component: ComboButton,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['sm', 'md', 'lg'],
+    },
+    // }
+    modifiers: {
+      control: {
+        type: 'select',
+        labels: Object.keys(modifiers),
+      },
+      options: Object.values(modifiers).map((_k, i) => i),
+      mapping: Object.values(modifiers),
```

Comment:
If you update with the format I suggest, then this should also be replaced with `children`.